src/dispatch/plugins/dispatch_slack/incident/interactive.py

```diff
@@ -580,7 +582,7 @@ async def handle_list_tasks_command(
                     accessory=Button(
                         text=button_text,
                         value=button_metadata,
-                        action_id=TaskNotificationActionIds.update_status,
+                        action_id=ConversationButtonActions.update_task_status,
```

Comment:
I'd revert this change and then make `TaskNotificationActionIds.update_status` equal to `ConversationButtonActions.update_task_status` in the `enums.py` module